src/Clb.hs

```diff
@@ -286,7 +311,7 @@ txOutRefAt addr = gets (txOutRefAtState $ C.toShelleyAddr addr)
 
 -- | Read all TxOutRefs that belong to given address.
 txOutRefAtState :: L.Addr L.StandardCrypto -> ClbState -> [P.TxOutRef]
-txOutRefAtState addr st = txInfoIn' <$> M.keys (M.filter atAddr utxos)
+txOutRefAtState addr st = transTxIn <$> M.keys (M.filter atAddr utxos)
```

Comment:
AFAIU, they are used only on side of Atlas. So could not they be moved out of Clb then?

Reply:
No objections, will move it out to Atlas.